pylutron/__init__.py

```diff
@@ -894,20 +897,15 @@ def name(self):
   def state(self):
     """Returns the current occupancy state."""
     # Poll for the first request.
-    ev = self._query_waiters.request(self._do_query_state)
-    ev.wait(1.0)
+    if self._state == OccupancyGroup.State.UNSET:
+      ev = self._query_waiters.request(self._do_query_state)
+      ev.wait(1.0)
```

Comment:
ah, this answers the other questions, was looking one at a time. Could you squash these two commits?